sms-sending/config.yml

```diff
@@ -0,0 +1,47 @@
+---
+title: Auto Send SMS
+description: Auto send SMS after a call gets disconnected using Platform API.
+categories:
+- Architect
+steps:
+- title: Introduction
+  content: |
+    The main objective of this tutorial is to showcase how to create an interaction, and send SMS using through a queue with Platform API. The given sample code will send a message after an agent drops off a call. This tutorial will also cover how to listen to a topic from a pre-created channel.
+- title: Authenticate with PureCloud
+  content: |
+    First, authenticate the application with pureCloud.
+- title: WebSocket Object
+  content: |
+    Create an instance of the web socket object that will listen to the topic {"id": "v2.routing.queues.{queueID}.conversations.calls"}. For convenience, a notification channel subscribed to a queues call flow, is already created. For more information about Notification API refer the following resources below:
+    * [Notifications](https://developer.mypurecloud.com/api/rest/v2/notifications/)
```

Comment:
Don't include the domain here, just the path like this: `[Notifications](/api/rest/v2/notifications/)`